exporter/collector/traces.go

```diff
@@ -119,6 +123,22 @@ func NewGoogleCloudTracesExporter(ctx context.Context, cfg Config, version strin
 	return &TraceExporter{texporter: exp}, nil
 }
 
+func mappingFuncFromAKM(akm []AttributeKeyMapping) func(attribute.Key) attribute.Key {
```

Comment:
```suggestion
func (akm []AttributeKeyMapping) mapKey(attribute.Key) attribute.Key {
```

It's a bit more readable here as well as at initialization time.